recipes/optional-lite/all/conanfile.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+from conans import ConanFile, CMake, tools
+
+
+class OptionalLite(ConanFile):
+    name = "optional-lite"
+    
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/martinmoene/optional-lite"
+    description = "A single-file header-only version of a C++17-like optional, a nullable object for C++98, C++11 and later"
+    author = "Martin Moene"
+    topics = ("conan", "cpp98", "cpp17", "optional", "optional-implementations")
+    license = "Boost Software License - Version 1.0. http://www.boost.org/LICENSE_1_0.txt"
```

Comment:
Better use SPDX names for licenses.

`BSL-1.0` https://spdx.org/licenses/BSL-1.0.html